src/ir/function.rs

```diff
@@ -150,7 +150,7 @@ impl FunctionSig {
             CXCursor_CXXMethod => {
                 // For CXCursor_FunctionDecl, cursor.args() is the reliable way
                 // to get parameter names and types.
-                cursor.args()
+                cursor.args().expect("It cannot be None because we are in a method/function")
```

Comment:
So I'd expect we might want to test this with stuff like variadic functions, but if all tests are passing seems like we can keep it, and then if it fails in some weird situation returning an empty vector in the case it's none to preser behavior is a really easy change.